codechecker_lib/analyzers/result_handler_plist_to_file.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+import ntpath
+import os
+import shutil
+from abc import ABCMeta
+
+from codechecker_lib import plist_parser
+from codechecker_lib.logger import LoggerFactory
+from codechecker_lib.analyzers.result_handler_base import ResultHandler
+
+LOG = LoggerFactory.get_new_logger('PLIST TO FILE')
+
+
+class PlistToFile(ResultHandler):
+    """
+    Result handler for copying a plist file to a different location.
+    """
+
+    __metaclass__ = ABCMeta
```

Comment:
Subclassing support. The other _result handler_ classes are written the same way.